lib/msf/core/payload/pingback.rb

```diff
@@ -0,0 +1,35 @@
+# -*- coding => binary -*-
+
+require 'msf/core'
+require 'msf/core/module/platform'
+require 'rex/text'
+
+#
+# This class provides methods for calculating, extracting, and parsing
+# unique ID values used by payloads.
+#
+class Msf::Payload::Pingback
+
+
+  #
+  # Constants
+  #
+
+  def self.generate_raw(opts={})
```

Comment:
Where is the `generate_raw` method used?

Reply:
I dropped a breakpoint and I'm not seeing it being hit.  Removing the method also seems to have no effect.  I'll take it out, but I'm tagging @bwatters-r7 here as a sanity check.